### DISCUSSION TITLE
Has anyone successfully train models/process images with RTX 3090?

### DISCUSSION THREAD

**Discussion by elai125** [neuropoly/axondeepseg#691] [Help needed]
I have been encountering problems with RTX 3090. Just want to know if anyone has successfully trained models/processed images with RTX 3090. Any input is appreciated!

**Comment by hermancollin** [comment]
@elai125 I would suggest opening an issue on the [ivadomed repo](www.github.com/ivadomed/ivadomed), where they have more technical knowledge on how to handle the pytorch dependency for an edge case like yours.
